docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '3.4'
+
+services:
+  ai-battle-website:
+    build: .
+    ports:
+      - "8080:8080"
+    environment:
+      - USER=vagrant
```

Comment:
Configured this to match the value in https://github.com/JSJitsu/ai-battle-website/blob/master/bootstrap-docker.sh#L4 so that users can simply copy the configuration template.

Reply:
Would you mind adding the local code as a volume so that `docker-compose up` can be used for development? Thanks!

Reply:
Not at all, I'll take a look into it tonight.